h/services/annotation_moderation.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from h import models
+
+
+class AnnotationModerationService(object):
+    def __init__(self, session):
+        self.session = session
+
+    def create(self, annotation):
+        """
+        Create an annotation nipa flag.
```

Comment:
This docstring needs an update. More generally, it might not be a bad idea to name the service's methods in a way that's more obvious for _consumers_ of the service, rather than just descriptive of what happens internally. Specifically, I'm thinking something like `.hide(annotation)`. What do you reckon?